scripts/docsascode/buildDoc.sh

```diff
@@ -76,10 +76,16 @@ function build_doc {
           # merge new config on previous
           mergeYml $current_output_template_path/config.yml $current_output_template_path/yaml_tmp/config.yml
           mergeYml $current_output_template_path/dac_custom-theme.yml $current_output_template_path/yaml_tmp/dac_custom-theme.yml
-          
+
           # remove temp yaml folder
           rm -rf $current_output_template_path/yaml_tmp/
 
+          # Check if any fonts dir exists in the template and add any ttf file from it
+          if [ -d $output_path/fonts/ ]
```

Comment:
subfolder must be used only for theme inheritance. So ttf files must be located directly the $output_path.

Reply:
ok